<?xml version="1.0"?>
<launch>

  <node pkg="tf" type="static_transform_publisher" name="map_broadcaster"
        args="1 0 0 0 0 0 world map 10" />

  <node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen"
        args="-d $(find tams_phasespace)/config/LineIteractiveMarker.rviz" />

  <node pkg="tams_phasespace" name="LineIteractiveMarker" type="LineIteractiveMarker" output="screen" />

</launch>
